assets/dev/js/editor/document/save/hooks/ui/save/after.js

```diff
@@ -28,24 +28,25 @@ export class FooterSaverAfterSave extends HookUIAfter {
 	}
 
 	onPageStatusChange( newStatus ) {
-		if ( 'publish' === newStatus ) {
-			const buttons = [];
+		// Don't add the "Have a look" link in the theme builder.
+		if ( elementor.config.document.support_site_editor || 'publish' !== newStatus || ! elementor.config.document.urls.have_a_look ) {
```

Comment:
Move to hook conditions.

Reply:
Are you sure? @matipojo 
These conditions only apply to the toaster, not the rest of the things in the `apply()` method